debian/changelog

```diff
@@ -1,3 +1,21 @@
+opx-platform-config (1.6.0) unstable; urgency=medium
+
+  * Update: Added media configuration files
+  * Update: Added Dell EMC S5248F-ON
+  * Update: Added bcm_env.cfg with 32M dmasize for S5248-ON
+  * Update: Updated copyrights
+  * Update: Added ACL slice configuration to SAI initialization
+  * Update: Added start type determination
+  * Update: Refactored PSU fan speed mapping
+  * Update: Added support for external sensors in PAS
+  * Update: Updated SAI configuration for Dell EMC S4248FB(L)-ON
+  * Update: Updated SAI phy media and ACL profiles for Dell EMC S5148F-ON
+  * Update: Fix for Dell EMC S6010-ON LEDs
```

Comment:
These last 3 lines should be "Bugfix" entries.

Reply:
thank you, changed it.